### DISCUSSION TITLE
Thank Lissy93, Happy holidays

### DISCUSSION THREAD

**Discussion by EVOTk** [Lissy93/dashy#381] [General]
Thanks for all you've done.  ;-)

Thank you for providing us with Dashy and also for your follow-up (bug report, request for functions, ...)

Happy holidays !
![happy-holidays-search-engine-people](https://user-images.githubusercontent.com/45015615/147388596-2bf0b90d-a193-4693-a720-f1b9157e134d.jpg)

**Comment by lissy93** [comment]
Thanks so much @EVOTk, this means a lot :)
Thank you too for your work in finding bugs, translating to French, and being patient during development. I don't think Dashy would be anything without all the contributions from users like you, it's always very much appreciated :)

I hope you've had a very happy Christmas break!
